tensorflow/python/eager/pywrap_gradient_exclusions.cc

```diff
@@ -294,7 +294,7 @@ bool OpGradientDoesntRequireInputIndices(
           {"SparseSegmentSumWithNumSegments", {false, {3}}},
           {"SparseSlice", {false, {2, 4}}},
           {"SparseSoftmax", {false, {1}}},
-          {"SparseSoftmaxCrossEntropyWithLogits", {true, {}}},
+          {"SparseSoftmaxCrossEntropyWithLogits", {false, {1}}},
```

Comment:
Should be `{0}`?

Reply:
It uses op.inputs[0], and isn't this a "drop indices" list? So of [0, 1] we should drop 1 if we're using 0.

Reply:
Logits are required and labels are not, so to me it still seems like it should be `{1}`

Reply:
My bad. You are both correct. Sorry for the confusion.